packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -233,12 +238,30 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
         return (
             <button
                 className={this.props.managedClasses.pane_resizeHandle}
-                aria-hidden={true}
                 onMouseDown={this.onMouseDown}
+                onKeyDown={this.onKeyDown}
+                aria-hidden={true}
             />
         );
     }
 
+    /**
+     * Handle keyPress
+     */
+    public onKeyDown = (e: React.KeyboardEvent<HTMLButtonElement>): void => {
+        switch (e.keyCode) {
+            case KeyCodes.arrowLeft /* tslint:disable-next-line */:
+                this.setWidth(this.rootElement.current.clientWidth - 1);
```

Comment:
Maybe make the "baseResizeIncrement" a const?  I could also see it as a something that someone might want to configure in props.

Reply:
I’m open to adding this as a feature down the road if requested, but would prefer to hold off for now. Incrementing by 10 with shift is a common pattern and this seems relatively easy to add modifications for if requested down the road (with a default to 10).